apps/web-client/src/lib/schemas/index.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+import type { Infer } from "sveltekit-superforms";
+
+export type SettingsSchema = Infer<typeof settingsSchema>;
```

Comment:
Move this back to `../forms/schema.ts`

Reply:
why? importing from forms causes a loading error on orders/supplier routes

Reply:
Because then we have two schema files. It works if its imported via `$lib/forms/schemas` (so the export from `forms/index` can be removed)